Pod/Classes/WPMediaCapturePreviewCollectionView.m

```diff
@@ -110,10 +110,19 @@ - (void)startCapture
 - (void)deviceOrientationDidChange:(NSNotification *)notification
 {
     if (self.captureVideoPreviewLayer.connection.supportsVideoOrientation) {
-        self.captureVideoPreviewLayer.connection.videoOrientation = (AVCaptureVideoOrientation)[[UIDevice currentDevice] orientation];
+        self.captureVideoPreviewLayer.connection.videoOrientation = [self videoOrientationForDeviceOrientation:[[UIDevice currentDevice] orientation]];
     }
 }
 
+- (AVCaptureVideoOrientation)videoOrientationForDeviceOrientation:(UIDeviceOrientation)orientation
+{
+    if (orientation == UIDeviceOrientationFaceUp || orientation == UIDeviceOrientationFaceDown) {
+        return AVCaptureVideoOrientationPortrait;
+    }
+
+    return (AVCaptureVideoOrientation)orientation;
```

Comment:
`AVCaptureVideoOrientation` can't be directly mapped to `UIDeviceOrientation`, as their values are different.